controllers/newsController.js

```diff
@@ -0,0 +1,32 @@
+const db = require("../models/index");
+const { Entry } = db;
+
+const Newscontroller = async function (req, res) {
+    // Save id from params 
+    const { idNews } = req.params;
+    // Req.Body for updates
+    const { name, content, image, type } = req.body;
+    // Find de element to update by primary Key
+    const newUpdate = await Entry.findByPk(idNews);
+
+    // Validate if element exists
+    if (newUpdate === null) {
+        return res.status(404).json({
+            success: false,
+            msj: 'No se encontró ninguna noticia',       
+        });
+    } else {
+        const updatedNews = await Entry.update({
```

Comment:
Dos preguntas aca:
Si ya tenes el entry, porque no lo modificas y haces un .save?

En vez de hacer el findByPk, porque no haces directamente el update y en base al resultado te fijas si actualizo o si no existe?

Reply:
La primer pregunta no se si la estoy entendiendo pero entiendo que está atada a la segunda.

Entonces vos lo que sugerís es que antes de chequear si existe directamente hacer el update guardandolo en una variable. Hecho eso valido si el elemento existe o no, y en base a eso retorno una respuesta?

**Posible modificación:** 
1-Borré el findByPk 
2- Pasé a que haga el update. 
3- Valido si existe y retorno respuesta.

`const db = require("../models/index");
const { Entry } = db;

const UpdateNews = async function (req, res) {
    // Save id from params 
    const { idNews } = req.params;
    // Req.Body for updates
    const { name, content, image, type } = req.body;

    // Update the element
    const ModifyNew = await Entry.update({
        name,
        content,
        image,
        type
    },{ returning: true, where: { id: idNews } })

    // Validate if element exists
    if (ModifyNew === null) {
        return res.status(404).json({
            success: false,
            msj: 'No se encontró ninguna noticia',       
        });
    } else {  
        res.status(200).send(`La noticia Nº${idNews} fue modificada. Actualizado con: name: ${name} content: ${content} image: ${image} type: ${type}`);
    }
}`

Reply:
Me gusta, espero a que lo agregues en el PR